packages/esm-patient-immunizations-app/src/immunizations/immunizations-sequence-table.tsx

```diff
@@ -26,23 +26,27 @@ const SequenceTable: React.FC<SequenceTableProps> = ({ immunizations }) => {
 
   const launchPatientImmunizationForm = useCallback(
     (immunizationFormData: Immunization, existingDoses: ExistingDoses) => {
-      const { vaccineName, vaccineUuid, sequences } = immunizationFormData;
-      const { sequenceLabel, sequenceNumber } = existingDoses;
-      const formHeader = t('immunizationForm', 'Immunization Form');
-      openWorkspaceTab(ImmunizationsForm, formHeader, {
-        vaccineName,
-        vaccineUuid,
-        immunizationObsUuid: existingDoses.immunizationObsUuid,
-        manufacturer: existingDoses.manufacturer,
-        lotNumber: existingDoses.lotNumber,
-        expirationDate: existingDoses.expirationDate,
-        sequences,
-        currentDose: {
-          sequenceLabel,
-          sequenceNumber,
-        },
-        vaccinationDate: existingDoses?.occurrenceDateTime,
-      });
+      launchPatientWorkspace('immunization-form-workspace');
+
+      // What's all this about?
```

Comment:
@Eriq-Kibet @denniskigen @donaldkibet Do any of you know what to do with this `openWorkspaceTab` usage? It's the only one in the codebase. Is this dead code or what?

Reply:
Thanks @donaldkibet . Can you take care of doing that?